### PR TITLE
Nl 59 defining user settings alice

### LINKED ISSUE #59: defining user settings
## Name and Email
  - Name: Dustin Lin
  - Email: d6lin@ucsd.edu

## Complexity of Issue (Scale from 1-5)
  - 2
  
## Issue Description
  - the front and backend should communicate what user settings will be defined to store them in the database and create crud functions

### LINKED ISSUE #84: User profile picture
## Issue Description
  - pull their profile picture from their firebase/google account and display
  - need to coordinate with frontend team on where to place it

### PR DESCRIPTION
**Pull request by Yenyun035**
Now this branch should have these features working:
- R and U for banner image
- logout
- reset password
- U for theme (only manually tested on Weekly overview. Need to make sure other pages only have their theme color updated)

### CODE REVIEW

**Comment by Rick10101221** [comment]
There's a scrollbar on the Weekly page. Is there a way we could remove this?
image

**Comment by s1hu** [comment]
Profile image support implemented

**Comment by DustinLin** [comment]
> There's a scrollbar on the Weekly page. Is there a way we could remove this? image

I think this is happening because the middle column is larger, issue #79

File: source/Login/WeeklyOverview/WeeklyStyles.css [hunk 1]
```diff
@@ -217,7 +217,7 @@ span {
 /* css for settings */
 #settings {
     background-color: #ffffff;
-    width: 242px;
+    width: 280px;
```
Comment (DustinLin) [inline source/Login/WeeklyOverview/WeeklyStyles.css:248]: Not sure if we should be hard coding these values, especially the widths of elements

Comment (Yenyun035) [reply source/Login/WeeklyOverview/WeeklyStyles.css:248]: @s1hu Better to use relative units

Comment (DustinLin) [reply source/Login/WeeklyOverview/WeeklyStyles.css:248]: discussed in meeting changes, leaving for now

**Pull request merged**